server/schedule/operator/create_operator.go

```diff
@@ -162,12 +162,13 @@ func CreateMergeRegionOperator(desc string, cluster opt.Cluster, source *core.Re
 
 	brief := fmt.Sprintf("merge: region %v to %v", source.GetID(), target.GetID())
 	op1 := NewOperator(desc, brief, source.GetID(), source.GetRegionEpoch(), kind|OpMerge, steps...)
-	op2 := NewOperator(desc, brief, target.GetID(), target.GetRegionEpoch(), kind|OpMerge, MergeRegion{
+	op2 := NewOperator(desc, brief, target.GetID(), target.GetRegionEpoch(), kind&OpAdmin|OpPlaceholder, MergeRegion{
```

Comment:
Why `&OpAdmin`?

Reply:
This operator can only be `(OpAdmin, OpPlaceholder)` or `(OpPlaceholder)`

Reply:
Why not directly use `OpPlaceholder`?